src/dos/cdrom_image.cpp

```diff
@@ -38,12 +43,32 @@
 #include <string.h>
 #endif
 
+#if defined(WORDS_BIGENDIAN)
+#define IS_BIGENDIAN true
+#else
+#define IS_BIGENDIAN false
+#endif
+
 using namespace std;
 
 #define MAX_LINE_LENGTH 512
 #define MAX_FILENAME_LENGTH 256
 
+#ifdef DEBUG
+char* get_time() {
```

Comment:
Formatting in this function needs to be fixed.

Reply:
Thanks, will make this change.